### PR TITLE
P20 AUVs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
image

File: calibration/FLORTN/CGINS-FLORTN-08364__20230703.csv
Comment (reedan88) [inline calibration/FLORTN/CGINS-FLORTN-08364__20230703.csv:1]: Looks good

File: deployment/CP16MOAS-A6263_Deploy.csv
Comment (reedan88) [inline deployment/CP16MOAS-A6263_Deploy.csv:1]: Looks good.

File: calibration/FLORTN/CGINS-FLORTN-08365__20230703.csv
Comment (reedan88) [inline calibration/FLORTN/CGINS-FLORTN-08365__20230703.csv:1]: Looks good.

File: deployment/CP16MOAS-A6264_Deploy.csv
Comment (reedan88) [inline deployment/CP16MOAS-A6264_Deploy.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]
I like how AUV 6264 was out for exactly 24 hours on-the-dot.

File: calibration/FLORTN/CGINS-FLORTN-08364__20230703.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+8364,CC_angular_resolution,1.076,
+8364,CC_depolarization_ratio,0.039,
+8364,CC_measurement_wavelength,700,
+8364,CC_scattering_angle,124,
```
Comment (sbeaulieu) [inline calibration/FLORTN/CGINS-FLORTN-08364__20230703.csv:5]: looks good

File: calibration/FLORTN/CGINS-FLORTN-08365__20230703.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+8365,CC_angular_resolution,1.076,
+8365,CC_depolarization_ratio,0.039,
+8365,CC_measurement_wavelength,700,
+8365,CC_scattering_angle,124,
```
Comment (sbeaulieu) [inline calibration/FLORTN/CGINS-FLORTN-08365__20230703.csv:5]: looks good

File: deployment/CP16MOAS-A6263_Deploy.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+AR82,,AR82,,CP16MOAS-A6263-01-FLORTN000,1,1,2024-04-06T12:33:00,2024-04-07T00:12:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-08364,35.93147,-75.3131,,,,across-shelf survey; AC-1 to AC-2,,
+AR82,,AR82,,CP16MOAS-A6263-02-DOSTAN000,1,1,2024-04-06T12:33:00,2024-04-07T00:12:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01519,35.93147,-75.3131,,,,across-shelf survey; AC-1 to AC-2,,
+AR82,,AR82,,CP16MOAS-A6263-03-CTDAVN000,1,1,2024-04-06T12:33:00,2024-04-07T00:12:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00425,35.93147,-75.3131,,,,across-shelf survey; AC-1 to AC-2,,
+AR82,,AR82,,CP16MOAS-A6263-04-NUTNRN000,1,1,2024-04-06T12:33:00,2024-04-07T00:12:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-01109,35.93147,-75.3131,,,,across-shelf survey; AC-1 to AC-2,,
+AR82,,AR82,,CP16MOAS-A6263-05-ADCPAN000,1,1,2024-04-06T12:33:00,2024-04-07T00:12:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,35.93147,-75.3131,,,,across-shelf survey; AC-1 to AC-2,,
+AR82,,AR82,,CP16MOAS-A6263-06-PARADN000,1,1,2024-04-06T12:33:00,2024-04-07T00:12:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,35.93147,-75.3131,,,,across-shelf survey; AC-1 to AC-2,,
```
Comment (sbeaulieu) [inline deployment/CP16MOAS-A6263_Deploy.csv:8]: looks good

File: deployment/CP16MOAS-A6264_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+AR82,,AR82,,CP16MOAS-A6264-01-FLORTN000,1,1,2024-04-06T17:55:00,2024-04-07T17:55:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-08365,36.17153,-74.96852,,,,along-shelf survey; AL-1 to AL-7,,
+AR82,,AR82,,CP16MOAS-A6264-02-DOSTAN000,1,1,2024-04-06T17:55:00,2024-04-07T17:55:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,36.17153,-74.96852,,,,along-shelf survey; AL-1 to AL-7,,
+AR82,,AR82,,CP16MOAS-A6264-03-CTDAVN000,1,1,2024-04-06T17:55:00,2024-04-07T17:55:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00424,36.17153,-74.96852,,,,along-shelf survey; AL-1 to AL-7,,
+AR82,,AR82,,CP16MOAS-A6264-04-NUTNRN000,1,1,2024-04-06T17:55:00,2024-04-07T17:55:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-01121,36.17153,-74.96852,,,,along-shelf survey; AL-1 to AL-7,,
+AR82,,AR82,,CP16MOAS-A6264-05-ADCPAN000,1,1,2024-04-06T17:55:00,2024-04-07T17:55:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,36.17153,-74.96852,,,,along-shelf survey; AL-1 to AL-7,,
+AR82,,AR82,,CP16MOAS-A6264-06-PARADN000,1,1,2024-04-06T17:55:00,2024-04-07T17:55:00,CGVCP-16MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50372,36.17153,-74.96852,,,,along-shelf survey; AL-1 to AL-7,,
```
Comment (sbeaulieu) [inline deployment/CP16MOAS-A6264_Deploy.csv:8]: looks good

**Review by sbeaulieu — APPROVED** [review]

**Pull request merged**